src/VisualStudio/Core/Def/TableDataSource/Suppression/VisualStudioDiagnosticListSuppressionStateService.cs

```diff
@@ -200,16 +182,6 @@ private static bool IsNonRoslynEntrySupportingSuppressionState(ITableEntryHandle
     private static bool IsEntryWithConfigurableSuppressionState([NotNullWhen(true)] DiagnosticData? entry)
         => entry != null && !SuppressionHelpers.IsNotConfigurableDiagnostic(entry);
 
-    private static AbstractTableEntriesSnapshot<DiagnosticTableItem>? GetEntriesSnapshot(ITableEntryHandle entryHandle, out int index)
-    {
-        if (!entryHandle.TryGetSnapshot(out var snapshot, out index))
-        {
-            return null;
-        }
-
-        return snapshot as AbstractTableEntriesSnapshot<DiagnosticTableItem>;
```

Comment:
this 'as' must fail, as there are no actual instances of this type.  So thsi method collapsed to `return null;` which i then inlined to callsites.